app/models/appearance.rb

```diff
@@ -0,0 +1,9 @@
+class Appearance < ActiveRecord::Base
```

Comment:
This is my join model.  The principle advantage of a `has_many :through` is that you can _represent the association as a meaningful thing_.  In this case a given `Appearance` has an arc.  That's pretty cool!